packages/techdocs-common/src/stages/publish/helpers.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export type supportedFileType = 'html' | 'css';
```

Comment:
Do we not currently support image/other assets?

Reply:
Aah, Sorry, this is a bad choice of naming from my side! 

We need to set specific `Content-Type` headers to serve some file types - `html` and `css` are those. We do support more kinds of file extensions. I'm thinking to remove this type since it causes confusion.

But this made me think that we can add another layer of security here in `techdocs-backend` plugin by creating an "allowedList" (to filter out JS files explicitly). We already remove `<script>` tags in the frontend side i.e. TechDocs plugin. (Will follow up with an issue).